analytics_data_api/v0/serializers.py

```diff
@@ -452,11 +450,17 @@ def get_engagement_ranges(self, obj):
         for entity_type in engagement_entity_types.AGGREGATE_TYPES:
             for event in engagement_events.EVENTS[entity_type]:
                 metric = '{0}_{1}'.format(entity_type, event)
+                # It's assumed that there may be any combination of low, normal,
```

Comment:
So yes, you're right. I'm going to change the test. I think what we decided is that whatever the DB exposes for ranges is what we expose in the API.